src/components/Header/Header.tsx

```diff
@@ -114,11 +114,9 @@ const Header = () => {
       setPollForAddress(false);
       let newAddress = "";
       if (wallet) {
-        const response = await wallet.getChangeAddress().catch((e: any) => console.error('Failed to get change address:', e));
-        newAddress = Address.from_bytes(
-          Buffer.from(response, "hex")
-        ).to_bech32();
+        newAddress = (await getAddresses(wallet))[2];
```

Comment:
`newAddress` would have the `changeAddress` in it, and it would be different each time the wallet is used - right?

In that case, when logged into a yoroi wallet in the tool, and reloading the page; will it retain the logged in state? Can you ensure this

Reply:
You are right, good catch. In the beginning it was like that but later when I realised we need both change address and stake one I changed to this.